lib/src/showcase_widget.dart

```diff
@@ -33,6 +33,7 @@ class ShowCaseWidget extends StatefulWidget {
   final Duration autoPlayDelay;
   final bool autoPlayLockEnable;
   final bool disableAnimation;
+  final bool disableOverlayClick;
```

Comment:
A better name than `disableOverlayClick` would be good. What happens on the overlay click? is it dismissed? or the next is displayed?

Reply:
I suppose its dismiss. So a better name would be `dismissable`.

Reply:
@BirjuVachhani  : it is used for disable next click when click on overlay.

Reply:
@MehulKK So its like go to next if any or dismiss right? Maybe we could say `disableBarrierInteractions` because its more like how flutter refers to this things! Barrier!

Reply:
@BirjuVachhani  :  ok, let me update name.